README.md

```diff
@@ -1,67 +1,79 @@
+1. Clone this repository, and `cd` into it
+1. Target the space you want to deploy the broker to
 
-    Note: the following credentials must be available in the `cdn-creds` service or in top-level environment variables:
+    ```bash
+    $ cf target -o <org> -s <space>
+    ```
 
-    * PORT
-    * BROKER_USER
-    * BROKER_PASS
-    * DATABASE_URL
-    * EMAIL
-    * ACME_URL
-    * BUCKET
-    * AWS_ACCESS_KEY_ID
-    * AWS_SECRET_ACCESS_KEY
-    * AWS_DEFAULT_REGION
+1. Create a [user-provided service](http://docs.cloudfoundry.org/devguide/services/user-provided.html) with the configuration (you will be prompted for the values of each)
 
-    ```
+    ```bash
     $ cf create-service rds micro-psql cdn-rds
-    $ cf create-user-provided-service cdn-creds -p '{"AWS_ACCESS_KEY_ID": "[key-id]", ...}'
+    $ cf create-user-provided-service cdn-creds -p 'BROKER_USER,BROKER_PASS,DATABASE_URL,EMAIL,ACME_URL,BUCKET,AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY,AWS_DEFAULT_REGION'
     ```
 
-* Deploy application
+1. Deploy the broker as an application
 
-    ```
+    ```bash
     $ cf push
     ```
 
-* Add to Cloud Foundry
+1. [Register the broker](http://docs.cloudfoundry.org/services/managing-service-brokers.html#register-broker)
 
-    ```
+    ```bash
     $ cf create-service-broker cdn-route [username] [password] [app-url] --space-scoped
     ```
 
 ## Usage
 
-* Create service
+1. Target the space your application is running in
 
+    ```bash
+    $ cf target -o <org> -s <space>
     ```
+
+1. Create service instance
+
+    ```bash
     $ cf create-service cdn-route cdn-route my-cdn-route \
         -c '{"domain": "my.domain.gov", "origin": "my-app.apps.cloud.gov"}'
 
     Create in progress. Use 'cf services' or 'cf service my-cdn-route' to check operation status.
     ```
 
-* Get CNAME instructions
+1. Get CNAME instructions
 
-    ```
+    ```bash
     $ cf service my-cdn-route
 
     Last Operation
     Status: create in progress
     Message: Provisioning in progress; CNAME domain "my.domain.gov" to "d3kajwa62y9xrp.cloudfront.net."
     ```
 
-* Update CNAME
+1. Create/update your CNAME in your DNS configuration
+1. Create a [private domain](http://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html#private-domains) in your Cloud Foundry organization
+
+    ```bash
+    $ cf create-domain <org> <domain>
+    ```
+
+1. [Add the `domain` to your manifest](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#domain) to connect that domain to your application, then `cf push`.
+    1. If you aren't using a manifest (which you should), [create a route manually](http://docs.cloudfoundry.org/devguide/deploy-apps/routes-domains.html#create-route-without-hostname).
 
-* Wait for changes to propagate (may take 30 minutes)
+        ```bash
+        $ cf create-route <space> <domain>
+        $ cf map-route <app> <domain>
+        ```
 
-* Visit `my.domain.gov`
+1. Wait for DNS changes to propagate (may take 30 minutes)
```

Comment:
At least in my testing (using short ttls), the thing that takes up to 30 minutes is cloudfront provisioning. I don't want users to expect this to work as soon as the dns changes propagate.